src/tests/test_cases/noqa.py

```diff
@@ -16,7 +16,8 @@ def docstring_bad_ignore_one():  # noqa: D400,D401,D415
     pass
 
 
-@expect("D401: First line should be in imperative mood ('Run', not 'Runs')")
+@expect("D401: First line should be in imperative mood "
+        "(perhaps 'Run', not 'Runs')")
```

Comment:
Seeing as the bug was caused by two different imperative forms of the same stem, I was indeed going for a less confident recommendation. 

Some options here:

1. perhaps 'Run', not 'Runs'
2. 'Run', not 'Runs'
3. e.g., 'Run' instead of 'Runs'
4. perhaps 'Run' instead of 'Runs'

I chose (1) because it's more clear that 'Runs' is *definitely* bad while 'Run' is *not necessarily* the right choice. 

@shacharoo LMKWYT

Reply:
I'm merging so that other PRs are unblocked, I think the current phrasing is fine for now, but we can revisit it before the next release.

Reply:
Thanks. (I like 3 and 4 for what it's worth).